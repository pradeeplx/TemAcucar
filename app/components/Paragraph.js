import React from 'react'
import Colors from "../Colors"
import Sentence from "./Sentence"

export default Paragraph = ({ children }) => (
  <Sentence style={{
    alignSelf: 'stretch',
    marginBottom: 20,
  }}>
    {children}
  </Sentence>
)
