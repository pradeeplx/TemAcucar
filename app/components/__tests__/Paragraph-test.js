import 'react-native'
import React from 'react'
import Paragraph from '../Paragraph'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <Paragraph>FooBar</Paragraph>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
