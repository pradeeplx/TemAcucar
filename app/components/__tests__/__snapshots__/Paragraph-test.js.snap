exports[`test renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#654C50",
        "fontFamily": "Avenir",
        "fontSize": 18,
      },
      Object {
        "alignSelf": "stretch",
        "marginBottom": 20,
      },
    ]
  }>
  FooBar
</Text>
`;
